packages/botonic-react/src/webchat/header.jsx

```diff
@@ -75,19 +78,20 @@ export const DefaultHeader = props => {
   )
 }
 export const WebchatHeader = props => {
-  const { webchatState } = useContext(WebchatContext)
+  const { webchatState, useTheme } = useContext(WebchatContext)
+  const { theme } = webchatState
   const handleCloseWebchat = event => {
     props.onCloseClick(event.target.value)
   }
-  if (webchatState.theme.customHeader) {
-    let CustomHeader = webchatState.theme.customHeader
+  let CustomHeader = useTheme('header.custom')
```

Comment:
it's great that with useTheme the code is now much shorter 💯

Reply:
💃💃💃